security/stunnel/src/opnsense/service/templates/OPNsense/Stunnel/stunnel.conf

```diff
@@ -19,6 +19,9 @@ logId = unique
 [{{service['@uuid']}}]
 accept = {% if service.accept_address %}{{service.accept_address}}:{% endif %}{{service.accept_port}}
 connect = {% if service.connect_address.find(":") > -1 %}[{{service.connect_address}}]{% else %}{{service.connect_address}}{% endif %}:{{service.connect_port}}
+{% if service.protocol != 'none' %}
```

Comment:
use `not helpers.empty('service.protocol')` here after removing the none

Reply:
Ok, done. Interestingly this does not work - at least not the way I just commited.

`{% if not helpers.empty('service.protocol') %}` seems to always match, even if `service.protocol=smtp`

Reply:
`{% if service.protocol %}` works

Reply:
@fhloston I'm a bit lost here, `{% if not helpers.empty('service.protocol') %}` should match on all but empty values, right? (smtp is a valid option)

Reply:
I am at a loss aswell. Is using `{% if service.protocol %}` an issue? 

I see it is used similarily with the `service.accept_address` above.

Reply:
no, it's not, but in this case both should do the same. you do need to update the model version by the way (I think it's not in there yet)

Reply:
OK. I guess updating the plugin version now makes sense aswell.